_data/banking.yml

```diff
@@ -330,6 +330,14 @@ websites:
       software: Yes
       doc: https://www.firsttechfed.com/OnlineServices/bankonlinesecurely/SecurityKey
 
+    - name: FirstBank (Colorado)
```

Comment:
I really think that the `(Colorado)` needs to be removed. From their [about page](https://www.efirstbank.com/about-firstbank/about.htm),

> FirstBank now has over $17 billion in assets and over 120 locations in **Colorado, Arizona, and California**